### PR TITLE
Fixing broken logos

### PR DESCRIPTION
**Pull request by dannoh**
Fixing logos

### CODE REVIEW

**Comment by parnic** [comment]
#44

File: templates/MMM-NHL.njk [hunk 1]
```diff
@@ -94,7 +94,7 @@
                         {% if config.showLogos %}
                             <td>
                                 <img class="icon {{ "no-color" if not config.colored }}"
-                                    src="https://www-league.nhlstatic.com/images/logos/teams-20212022-dark/{{ series.teams.away.id }}.svg"/>
+                                    src="https://assets.nhle.com/logos/nhl/svg/{{games[index].teams.away.short}}_dark.svg"/>
```
Comment (parnic) [inline templates/MMM-NHL.njk:97]: These two should be `series.teams.[away|home].short`, not `games[index]`, fyi.

Comment (fewieden) [reply templates/MMM-NHL.njk]: @parnic good catch

**Comment by fewieden** [comment]
@dannoh thank you for your contribution

**Pull request merged**